packages/web-components/package.json

```diff
@@ -82,6 +82,7 @@
     "@carbon/layout": "10.19.0",
     "carbon-components": "10.29.0",
     "carbon-web-components": "1.13.0",
+    "js-cookie": "^2.2.1",
```

Comment:
@annawen1 would you be able to set up the cookie utility in this PR and move into that package?

Reply:
This is resolved in Hunter's PR: feat(cloud-masthead): add Cloud-specific container and composite components #5593